@xen-orchestra/web-core/lib/components/state-hero/VtsStateHero.vue

```diff
@@ -113,5 +117,24 @@ const imageSrc = computed(() => {
       width: 80%;
     }
   }
+
+  &.table {
+    padding: 5rem;
+    gap: 2.4rem;
+
+    .text {
+      order: 1;
+    }
+
+    .image {
+      order: 3;
+      max-height: 20rem;
+    }
+
+    .loader {
+      order: 2;
+      font-size: 10rem;
+    }
+  }
```

Comment:
From what i see in the DS, the text is under the image
![Capture d’écran de 2024-11-27 10-25-31](https://github.com/user-attachments/assets/f82f4c70-e39c-4611-8b72-20f7d89437da)